docs/guides/src/main/server/configuration.adoc

```diff
@@ -145,38 +139,35 @@ You can also specify a different configuration file by using the `[-cf|--config-
 <@kc.start rootParameters="-cf myconfig.conf"/>
 
 Changes to any *build option* defined in the `keycloak.conf` file that is targeted for the `build` command are ignored
-if the value differs from the value used to previously run the `build` command. In this case, make sure you run the `build` command again so that
+if the value differs from the value for the last `build` command. In this case, make sure you run the `build` command again so that
 any build option is updated accordingly.
 
-=== Understanding the development and production modes
-
-By default, the server defines two main operating modes:
+=== Development versus production mode
 
-* Development
-* Production
+The server supports the following operating modes:
 
-The development mode is activated every time you run the `start-dev` command. In this mode, some key configuration options are set to make it possible to start the
+Development mode:: This mode is activated every time you run the `start-dev` command. In this mode, some key configuration options are set to make it possible to start the
 server for development purposes without the burden of having to define additional settings that are mandatory for production.
 
-The production mode is activated by default when you run the `build` or the `start` command. Use this mode to set any configuration option that
+Production mode:: This mode is activated by default when you run the `build` or the `start` command. Use this mode to set any configuration option that
```

Comment:
```suggestion
Production mode:: This mode is activated when you run the `build` or the `start` command. Use this mode to set any configuration option that
```

Reply:
as we don't have profile support anymore, the "by default" can go away, as this is _the_ mode when invoking start / build now.